src/index.html

```diff
@@ -2,11 +2,237 @@
+      </div>
+      <div class="card" data-qa="card">
+        <img class="card__img" src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)">
+        <div class="card__title">
+          <p class="card__title-label">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+          <p class="card__title-code">Product code: 195434</p>
+        </div>
+        <div class="card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__reviews-counter">Reviews: 5</div>
+        </div>
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-amount">$2,199</p>
+        </div>
+        <a class="card__button" href="#">Buy</a>
+      </div>
+      <div class="card" data-qa="card">
+        <img class="card__img" src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)">
+        <div class="card__title">
+          <p class="card__title-label">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+          <p class="card__title-code">Product code: 195434</p>
+        </div>
+        <div class="card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__reviews-counter">Reviews: 5</div>
+        </div>
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-amount">$2,199</p>
+        </div>
+        <a class="card__button" href="#">Buy</a>
+      </div>
+      <div class="card">
+        <img class="card__img" src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)">
+        <div class="card__title">
+          <p class="card__title-label">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+          <p class="card__title-code">Product code: 195434</p>
+        </div>
+        <div class="card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__reviews-counter">Reviews: 5</div>
+        </div>
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-amount">$2,199</p>
+        </div>
+        <a class="card__button" href="#"
+        data-qa="nav-hover">Buy</a>
```

Comment:
This is breaking your tests.

Reply:
what exactly in this place breaking my test?